css/style.css

```diff
@@ -5,6 +5,13 @@ body {
   margin: 0;
 }
 
+.desk-item,
```

Comment:
- [ ] [REQUIRED] Kindly use media queries for the desktop version. Media queries are a key part of responsive web design, as they allow you to create different layouts depending on the size of the viewport, but they can also be used to detect other things about the environment your site is running on
![queries](https://user-images.githubusercontent.com/63964646/144637342-c758549b-423a-4edf-afe7-d937f4fadf51.PNG)
.

Reply:
Thank you for your constructive feedback you gave but I have used media queries for the desktop version. I kindly ask you to check my HTML header file. I have included there.

Reply:
> Thank you for your constructive feedback you gave but I have used media queries for the desktop version. I kindly ask you to check my HTML header file. I have included there.

Reply:
How about this media query. Have you checked it in the index.html file should I always supposed to use it in css file only.